README-pl-pl.md

```diff
@@ -0,0 +1,1744 @@
+  - [`Number.toFixed()` display different numbers](#numbertofixed-display-different-numbers)
+  - [`Math.max()` mniej niż `Math.min()`](#mathmax-mniej-niż-mathmin)
+  - [Comparing `null` to `0`](#comparing-null-to-0)
+  - [Redeklaracja tej samej zmiennej](#redeklaracja-tej-samej-zmiennej)
+  - [Domyślne zachowanie Array.prototype.sort()](#domyślne-zachowanie-arrayprototypesort)
+- [📚 Inne materiały](#-inne-materiały)
+- [🎓 Licencja](#-licencja)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- prettier-ignore-end -->
+
+# 💪🏻 Motywacja
+
+> Dla zabawy
+>
+> &mdash; _[**“Just for Fun: The Story of an Accidental Revolutionary”**](https://en.wikipedia.org/wiki/Just_for_Fun), Linus Torvalds_
+
+Głównym celem tej listy jest zebranie szalonych przykładów i wyjaśnienie, w jaki sposób działają, jeśli to możliwe. Tylko dlatego, że fajnie jest nauczyć się czegoś, czego wcześniej nie znaliśmy.
+
+Jeśli jesteś początkujący, możesz skorzystać z tych notatek, aby głębiej zagłębić się w JavaScript. Mam nadzieję, że te notatki zmotywują cię do spędzenia więcej czasu na czytaniu specyfikacji.
+
+Jeśli jesteś profesjonalnym programistą, możesz rozważyć te przykłady jako świetne źródło informacji o wszystkich dziwactwach i nieoczekiwanych krawędziach naszego ukochanego JavaScript.
+
+W każdym razie po prostu przeczytaj to. Prawdopodobnie znajdziesz coś nowego.
+
+# ✍🏻 Notacja
+
+**`// ->`** służy do wyświetlenia wyniku wyrażenia. Na przykład:
+
+```js
+1 + 1; // -> 2
+```
+
+**`// >`** oznacza wynik `console.log` lub wyświetlenie innego wyniku. Na przykład:
+
+```js
+console.log("hello, world!"); // > hello, world!
+```
+
+**`//`** jest tylko komentarzem używanym w celu wyjaśnienia. Przykład:
+
+```js
+// Assigning a function to foo constant
+const foo = function() {};
+```
+
+# 👀 Przykłady
+
+## `[]` jest równe `![]`
+
+Tablica jest równa zanegowanej tablicy:
+
+```js
+[] == ![]; // -> true
+```
+
+### 💡 Wytłumaczenie:
+
+Abstrakcyjny operator równości przekształca obie strony na liczby, aby je porównać, a obie strony stają się liczbą `0` z różnych powodów. Tablice są prawdziwe, więc po prawej stronie przeciwieństwem prawdziwej wartości jest `false`, który jest następnie wymuszany na `0`. Po lewej jednak pusta tablica jest wymuszana na liczbę, nie będąc najpierw wartością logiczną, a puste tablice są wymuszane na `0`, mimo że są prawdziwe.
+
+Oto jak to wyrażenie upraszcza:
+
+```js
++[] == +![];
+0 == +false;
+0 == 0;
+true;
+```
+
+Zobacz też [`[]` is truthy, but not `true`](#-is-truthy-but-not-true).
+
+- [**12.5.9** Logical NOT Operator (`!`)](https://www.ecma-international.org/ecma-262/#sec-logical-not-operator)
+- [**7.2.13** Abstract Equality Comparison](https://www.ecma-international.org/ecma-262/#sec-abstract-equality-comparison)
+
+## `true` nie jest równe `![]`, ale też nie równe `[]`
+
+Tablica nie jest równa `true`, ale nie Tablica nie jest równa `true` również;
```

Comment:
Change 'nie Tablica' to 'zanegowana tablica', it sounds better & more official.

Reply:
Good idea, thansk a lot!